.github/actions/support-package-metadata-json/metadata.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
```

Comment:
It might be a bit confusing to have scripts having to do with support packages in .github/actions, .github/support-packages, and kernel-modules/support-packages. Perhaps they should be put into one place or there should be README.md files that explain the different locations for scripts.

Reply:
With these changes, there are no more scripts under `.github/support-packages` and the scripts in the actions directory are there because they are specifically used with the action they are part of. I agree we need to make a better job of organizing our scripts, but IDK if this is the right PR to address this.

Reply:
Okay